app_flutter/test/commit_box_test.dart

```diff
@@ -71,5 +72,50 @@ void main() {
       // Image.Network throws a 400 exception in tests
       tester.takeException();
     });
+
+    testWidgets('tapping redirect button should redirect to Github',
+        (WidgetTester tester) async {
+      // The url_launcher calls get logged in this channel
+      const MethodChannel channel =
+          MethodChannel('plugins.flutter.io/url_launcher');
+      final List<MethodCall> log = <MethodCall>[];
+      channel.setMockMethodCallHandler((MethodCall methodCall) async {
+        log.add(methodCall);
+      });
+
+      await tester.pumpWidget(MaterialApp(
+        home: CommitBox(
+          commit: expectedCommit,
+        ),
+      ));
+
+      tester.takeException();
```

Comment:
Why are we ignoring exceptions here and below? Maybe add a comment with a quick explanation or (better:) add an expect to assert that the exception returned is actually what you expect it to be.

Reply:
Removed, it was left over :)